tests/jballerina-unit-test/src/test/resources/test-src/types/future/future_positive.bal

```diff
@@ -0,0 +1,180 @@
+function testBasicTypes() {
+    future<int> f1 = start add(5, 2);
+    future<boolean> f2 = start status();
+    future<string> f3 = start concat("foo");
+
+    int result1 = wait f1;
+    boolean result2 = wait f2;
+    string result3 = wait f3;
+
+    assertEquality(7, result1);
```

Comment:
Shall we add assertions for `result2` and `result3` also? Please check for all tests.

Reply:
Added